lua/nvim_lsp.lua

```diff
@@ -1,5 +1,30 @@
 local configs = require 'nvim_lsp/configs'
 
+require 'nvim_lsp/bashls'
+require 'nvim_lsp/ccls'
+require 'nvim_lsp/clangd'
+require 'nvim_lsp/cssls'
+require 'nvim_lsp/dockerls'
+require 'nvim_lsp/elmls'
+require 'nvim_lsp/flow'
+require 'nvim_lsp/fortls'
+require 'nvim_lsp/ghcide'
+require 'nvim_lsp/gopls'
+require 'nvim_lsp/hie'
+require 'nvim_lsp/intelephense'
+require 'nvim_lsp/julials'
+require 'nvim_lsp/leanls'
+require 'nvim_lsp/pyls'
+require 'nvim_lsp/pyls_ms'
+require 'nvim_lsp/rls'
+require 'nvim_lsp/rust_analyzer'
+require 'nvim_lsp/solargraph'
+require 'nvim_lsp/sumneko_lua'
+require 'nvim_lsp/texlab'
+require 'nvim_lsp/tsserver'
+require 'nvim_lsp/vimls'
+require 'nvim_lsp/ocamlls'
```

Comment:
@kdheepak 
Sorry, we don't need these codes anymore after #109.

Reply:
My bad! I messed up resolving the merge conflict.